easybuild/easyblocks/l/lammps.py

```diff
@@ -231,23 +236,32 @@ def extra_options(**kwargs):
         extra_vars['separate_build_dir'][0] = True
         return extra_vars
 
-    def update_kokkos_cpu_mapping(self):
+    def prepare_step(self, *args, **kwargs):
+        """Custom prepare step for LAMMPS."""
+        super(EB_LAMMPS, self).prepare_step(*args, **kwargs)
 
-        if LooseVersion(self.cur_version) >= LooseVersion(translate_lammps_version('31Mar2017')):
-            self.kokkos_cpu_mapping['neoverse_n1'] = 'ARMV81'
-            self.kokkos_cpu_mapping['neoverse_v1'] = 'ARMV81'
+        # version 1.3.2 is used in the test suite to check easyblock can be initialised
+        if self.version != '1.3.2':
+            self.cur_version = translate_lammps_version(self.version, path=self.start_dir)
+        else:
+            self.cur_version = self.version
+        self.ref_version = translate_lammps_version(ref_version)
 
-        if LooseVersion(self.cur_version) >= LooseVersion(translate_lammps_version('21sep2021')):
-            self.kokkos_cpu_mapping['a64fx'] = 'A64FX'
-            self.kokkos_cpu_mapping['zen4'] = 'ZEN3'
+        self.pkg_prefix = 'PKG_'
```

Comment:
Moving things from the `__init__` section runs the risk of breaking things with `--module-only`, have you verified that `--module-only` still works properly with these changes?

Reply:
Ah, it is in your original comment...I shut up now